lib/lure/repo.go

```diff
@@ -65,10 +78,16 @@ func cloneRepo(hgAuth Authentication, project Project) (Repo, error) {
 }
 
 func CheckForUpdatesJobCommand(auth Authentication, project Project, args map[string]string) error {
-	return checkForUpdatesJob(auth, project)
+	commitMessage, ok := args["commitMessage"]
+	if !ok {
```

Comment:
You could use the new method to initialize project that @fdeschodt did in the other PR, but we'll have to merge it first.